mathics/session.py

```diff
@@ -150,7 +150,7 @@ def evaluate(self, str_expression, timeout=None, form=None):
         )
         if form is None:
             form = self.form
-        self.last_result = expr.evaluate(self.evaluation)
+        self.last_result = expr.evaluate(self.evaluation) if expr else None
```

Comment:
In WMA, an empty expression should result in `Null` (`SymbolNull`). I think that what we should do here instead is to check in line 147 if   `expr` is None, and return `SymbolNull`.

Reply:
LGTM